msfits/MSFits/MSFitsOutput.cc

```diff
@@ -366,6 +366,22 @@ uInt MSFitsOutput::get_tbf_end(const uInt rownr, const uInt nrow,
     return tbfend;
 }
 
+static void defineRandomParam(Record& ek, Int n, const String& name) {
```

Comment:
Could you add some documentation to this function? In particular that what it writes is not 'random' (maybe rename to `defineArbitraryParam` or `defineAnyParam`?), and that it will write the scale to 1 and zero to 0.

Reply:
"random" is what FITS calls these parameters; I'll add a comment